src/mrpro/operators/FourierOp.py

```diff
@@ -223,3 +224,18 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor,]:
             x = x.permute(*unpermute)
 
         return (x,)
+
+    def __repr__(self):
+        """Representation method for Fourier Operator."""
+        device = str(self._traj.device)
+        if self._nufft_dims:
+            dims = self._nufft_dims
+        else:
+            dims = self._fft_dims
```

Comment:
a fourier op can have both.

it should be printed out in which dimensions nufft is done, in which dimensions fft is done and if cartesian rendering needs to be applied.

Reply:
aso recon matrix and image matrix

Reply:
and move the traj information to the end?

Reply:
Sorry, did not see the traj information comment, will address this in the next commit.